core/src/main/scala/spire/stats/Distribution.scala

```diff
@@ -0,0 +1,46 @@
+package spire.stats.distribution
+
+import spire.random.Generator
+
+trait Distribution[A] {
+
+  /** Function used to transform the values drawn from a Generator according to this probability distribution function
+    */
+  private[stats] val transform: (Generator => A)
```

Comment:
I guess it's more like a definition difference.
As seen in [1], a random variable is the set of outcomes from a random experiment, and the mathematical function describing the outcomes of the random variable and their associated probabilities is the (probability) distribution.
It's a composition relationship: a random variable has a probability distribution.
A probability distribution has a probability density function (or a probability mass function) and a cumulative distribution function. It can have location, scale or shape parameters, mean, etc.
The random variable has a distribution that you use to define it.

An example from [1]:
We can create a random variable `Y` that models a $1 payoff for a successful bet on heads as follows:

```
Y(w) = 1, if w=heads
       0, if w=tails
```

These are just the outcomes. Now, if the coin is a fair coin, `Y` has a probability mass function `f_Y` given by:

```
f_Y(y) = 1/2, if y=1
         1/2, if y=0
```

Also you can think about operating on random variables as: `Z = X + Y` but not on the distributions themselves [2].

My guess is that you are thinking: if the `transform` function is used only in the `RandomVariable[A]` why it's not defined there? Well, the reason I made it this way is that the `RandomVariable` just asks for whatever distribution it has to do the transformation since the distribution, when implemented, has all the information needed to do this. If we put it in the `RandomVariable` it would have to understand about the implementation.

[1] http://en.wikipedia.org/wiki/Random_variable
[2] http://math.stackexchange.com/questions/263974/sums-of-two-probability-density-functions